src/vs/workbench/api/node/extHostTerminalService.ts

```diff
@@ -318,6 +332,24 @@ export class ExtHostTerminalService implements ExtHostTerminalServiceShape {
 		return renderer;
 	}
 
+	public async createTerminalRendererForTerminal(terminal: vscode.Terminal): Promise<vscode.TerminalRenderer> {
```

Comment:
Based on discussion [here](https://github.com/Microsoft/vscode/issues/66818#issuecomment-461056194) this should be `resolveTerminalRenderer` and should take a terminal ID.